docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,554 @@
+  enabled: off
+proxy_service:
+  enabled: off
+ssh_service:
+  enabled: off
+app_service:
+  enabled: true
+  apps:
+  - name: azure-cli
+    cloud: Azure
+```
+
+Edit `/etc/teleport.yaml` to replace `teleport.example.com:3080` with the host
+and port of your Teleport Proxy Service or Teleport Cloud tenant, e.g.,
+`mytenant.teleport.sh:443`.
+
+The `app_service` field configures the Teleport Application Service. Each item
+within `app_service.apps` is an application configuration. 
+
+In this example, we have enabled Azure CLI access by setting `cloud` to `Azure`.
+With this setting configured, the Application Service will proxy user commands
+from the Azure CLI by requesting access to the Azure API under the user's chosen
+identity, which works as long as the identity is one of the ones attached to the
+Application Service host.
+
+### Run the Teleport Application Service
+
+On the host where you will run the Teleport Application Service, execute the
+following command, depending on whether you installed Teleport using a package
+manager or via a TAR archive:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem label="TAR archive">
+
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport; 
+$ sudo systemctl start teleport; 
+```
+</TabItem>
+</Tabs>
+
+## Step 3/4. Enable your user to access the Azure CLI
+
+The next step is to authorize your Teleport user to assume an Azure identity and
+execute Azure CLI commands via Teleport. You will protect access to this
+identity using Teleport's RBAC system, where a user's roles determine which
+Azure managed identities (if any) they can access.
+
+Create a file called `azure-cli-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: azure-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    azure_identities:
+      - /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/my-resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/teleport-azure
+
+```
+
+This role grants a user access to any Teleport-registered application, such as
+the `azure-cli` application we defined earlier, and allows that user to assume
+the `teleport-azure` identity you created earlier.
+
+Edit the identity URI in the `azure_identities` field to match the one you
+copied earlier.
+
+Create the role:
+
+```code
+$ tctl create -f azure-cli-access.yaml
+```
+
+Assign this role to your Teleport user by running the following commands,
+depending on whether you authenticate as a local Teleport user or via the
+`github`, `saml`, or `oidc` authentication connectors: 
+
+<Tabs>
+<TabItem label="Local User">
+
+Retrieve your local user's configuration resource:
+
+```code
+$ tctl get users/$(tsh status -f json | jq -r '.active.username') > out.yaml
+```
+
+Edit `out.yaml`, adding `azure-cli-access` to the list of
```

Comment:
Since we're using the new `add-role-to-user.mdx` partial, I've opened an issue to use `tctl users update` in that partial: https://github.com/gravitational/teleport/issues/19067